### PR TITLE
feat: implement seed word phrase counter

### PR DESCRIPTION
**Pull request by hydr063n**
fixes: https://github.com/status-im/nim-status-client/issues/1078

- If the count matches the required amount a tick will be included next the count
- The format is "✓ 12 words"
- This commit also fixes the text color when in dark in the wallet add new account modal
- This feature is added on both login and wallet add account with wallet seed modal

### CODE REVIEW

File: ui/imports/Utils.qml [hunk 1]
```diff
@@ -147,4 +147,24 @@ QtObject {
...
+            return 0;
+        return text.trim().split(" ").length;
+    }
+
+    /**
+     * Returns text in the format "✓ 12 words" for seed phrases input boxes
+     */
+    function seedPhraseWordCountText(text) {
+        let wordCount = countWords(text);
+        return getTick(wordCount) + wordCount.toString() + " words";
```
Comment (richard-ramos) [inline ui/imports/Utils.qml:168]:
```suggestion
        return qsTr("%1%2 words").arg(getTick(wordCount)).arg(wordCount.toString());
```

**Review by richard-ramos — CHANGES_REQUESTED** [review]
Very cool! only a small change requested for i18n and we're good to go! thank you, @hydr063n! :)

File: ui/imports/Utils.qml (hunk 1 above)
Comment (hydr063n) [reply ui/imports/Utils.qml]: Fixed! Forgot about translations!

**Review by jrainville — APPROVED** [review]
Nice, very cool

**Pull request merged**